tests/e2e/Services/Sites/SitesCustomServerTest.php

```diff
@@ -1563,7 +1563,154 @@ public function testSiteTemplate(): void
             'repository' => $template['providerRepositoryId'],
             'owner' => $template['providerOwner'],
             'rootDirectory' => $template['frameworks'][0]['providerRootDirectory'],
-            'version' => $template['providerVersion'],
+            'type' => 'tag',
+            'reference' => $template['providerVersion'],
+            'activate' => true
+        ]);
+
+        $this->assertEquals(202, $deployment['headers']['status-code']);
+        $this->assertNotEmpty($deployment['body']['$id']);
+
+        $deployment = $this->getDeployment($siteId, $deployment['body']['$id']);
+        $this->assertEquals(200, $deployment['headers']['status-code']);
+        $this->assertEquals(0, $deployment['body']['sourceSize']);
+        $this->assertEquals(0, $deployment['body']['buildSize']);
+        $this->assertEquals(0, $deployment['body']['totalSize']);
```

Comment:
This is nice, but afterwards, lets ensure they are greater than 0, after deployment finishes